ses

```diff
@@ -65,9 +65,18 @@ if [ -x /usr/bin/ceph ]; then
     # remain, because their output is easier to read in a hurry ;)
     plugin_command "ceph --connect-timeout=$CT report" > $LOG/ceph/ceph-report 2>&1
     plugin_command "timeout $CT rados df" > $LOG/ceph/rados-df 2>&1
-    plugin_command "timeout $CT rbd ls" > $LOG/ceph/rbd-ls 2>&1
     plugin_command "radosgw-admin period get" > $LOG/ceph/radosgw-admin-period-get 2>&1
 
+    timeout $CT rados lspools 2>/dev/null | sed -e '/.rgw./d' -e '/cephfs_/d' |
+    while read pools; do
+        mkdir -p $LOG/ceph/images/$pools
+        timeout $CT rbd -p $pools ls 2>/dev/null |
+        while read images; do
+            plugin_command "timeout $CT rbd -p $pools info $images" \
+                > $LOG/ceph/images/${pools}/rbd-info-${images} 2>&1
```

Comment:
Although this info may be very useful sometimes, for large clusters, with thousands of images running this will be too long.

I would just provide `rbd ls` output for all rbd pools. Alternatively you could add `OPTION_SES_RBD_INFO` option and run info only if it is set to true. Or provide `OPTION_SES_RBD_INFO_MAX` with the default = 20 and run `rbd info` for images not more than this number. See `OPTION_SES_INACTIVE_PG_QUERY_MAX` below as an example. I like the last option the most, as it is useful sometime to see info for a couple of images to get an idea e.g. about what features are used.

Reply:
That is why I wanted the info specifically as that is the more valuable information. But I agree it does not make sense getting this for 100s of Pools and images. What if we use the option you suggested as `OPTION_SES_RBD_INFO_MAX:=2` and get only the info for 2 images per rbd pool?

Reply:
I think `OPTION_SES_RBD_INFO_MAX:=10` (or 5) would be fine too